graphql/service.go

```diff
@@ -76,6 +78,7 @@ func (service *Service) RegisterInterface(t InterfaceDesc, impl InterfaceTypeRes
 func (service *Service) RegisterObject(t ObjectDesc, impl interface{}) {
 	cfg := t.Config()
```

Comment:
This call is now redundant, can we remove it from here and other similar places?